CRM/Contact/Form/Search/Custom/ContributionAggregate.php

```diff
@@ -234,10 +234,23 @@ public function where($includeContactIDs = FALSE) {
         }
         $clauses[] = "contrib.receive_date >= '{$value}'";
       }
+      elseif (strstr($name, '_high')) {
+        if (strlen($value) == 10) {
+          $value .= ' 23:59:59';
+        }
+        elseif (empty($value)) {
+          $value = date('Y-m-d H:i:s');
+        }
+
+        $clauses[] = "contrib.receive_date >= '{$value}'";
```

Comment:
seems wrong? in case of high, shouldn't it be `<=` instead of `>=`?

Reply:
Also, there is an else statement too, which will execute for relative dates, and it has `<=`. Maybe, this complete loop needs adjustments.

Reply:
Yes, I think i messed it up a bit, I'll send a new commit asap.

Thanks for the feedback @jitendrapurohit